### PR TITLE
HOTT-1870: Expose Origin reference document attributes on ROO API

### PR DESCRIPTION
**Pull request by maverickvi**
### Jira link

[HOTT-1870](https://transformuk.atlassian.net/browse/HOTT-1870)

### What?

I have added/removed/altered:

- [ ] Exposed Origin reference document attributes on Rules of origin API

### Why?

I am doing this because:

- We need to access Origin reference document attributes on the front end

### Have you? (optional)

- [ ] Added documentation for new apis

### Deployment risks (optional)

- Changes an api that is used in production

### CODE REVIEW

**Review by javier-transform — APPROVED** [review]
LGTM

File: app/models/rules_of_origin/origin_reference_document.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```
Comment (willfish) [inline app/models/rules_of_origin/origin_reference_document.rb:1]: You shouldn't need this - I suspect the wrong local rubocop config is being used

File: app/models/rules_of_origin/origin_reference_document.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+module RulesOfOrigin
+  class OriginReferenceDocument
+    include ActiveModel::Model
+
+    attr_accessor :ord_title, :ord_version, :ord_date, :ord_original
+    attr_writer :id
+
+    def id
+      @id = 'origin_reference_document_id'
```
Comment (willfish) [inline app/models/rules_of_origin/origin_reference_document.rb:11]: I suspect you want to use the ContentAddressableId module and specify the fields that make this entity unique

File: app/models/rules_of_origin/origin_reference_document.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module RulesOfOrigin
+  class OriginReferenceDocument
+    include ActiveModel::Model
+
+    attr_accessor :ord_title, :ord_version, :ord_date, :ord_original
+    attr_writer :id
```
Comment (willfish) [inline app/models/rules_of_origin/origin_reference_document.rb:8]: You don't want this

**Review by willfish — APPROVED** [review]

**Pull request merged**